tracing/lightstep.go

```diff
@@ -0,0 +1,52 @@
+package tracing
+
+import (
+	"go.opentelemetry.io/collector/translator/conventions"
+	"go.opentelemetry.io/otel/exporters/otlp"
+	"go.opentelemetry.io/otel/label"
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"google.golang.org/grpc/credentials"
+)
+
+// Lightstep service to export traces to
+type Lightstep struct {
+	Endpoint string `long:"lightstep-endpoint" description:"lightstep endpoint" default:"ingest.lightstep.com:443"`
```

Comment:
maybe `Address` is better, since that matches the `otlp.WithAddress` functional parameter?